libsqlite3-sys/Cargo.toml

```diff
@@ -37,6 +37,12 @@ wasm32-wasi-vfs = []
 [dependencies]
 openssl-sys = { version = "0.9", optional = true }
 
+[target.wasm32-unknown-unknown.dependencies]
```

Comment:
Should this include a `not(unix)` condition here because `js-sys` is a web-only dependency? `wasm32-unknown-unknown` doesn't always mean web. I did [something similar](https://github.com/1Password/sys-locale/blob/main/Cargo.toml#L15), along with a `js` feature and it's worked pretty well for quite a few downstream WASM users.

Reply:
It sounds like you might be conflating two different things here:

1) To my understanding, `wasm32-unknown-unknown` will never report itself as `unix`, so your proposed condition is unnecessary. `unix` *is* [set](https://github.com/rust-lang/rust/blob/1df0458781d6fd753a68c4cdc4de5313b1635dbd/compiler/rustc_target/src/spec/targets/wasm32_unknown_emscripten.rs#L22) by `wasm32-unknown-emscripten`, which is explicitly not matched by `[target.wasm32-unknown-unknown.dependencies]`, and is outside the scope of this PR.

2) The other thing is that `wasm32-unknown-unknown` doesn't always mean web, which is true. I think you actually meant to say that `wasm32-unknown-unknown` doesn't always mean a *Javascript environment*, which is also true. PR welcome for supporting that use case.